workspaces/acs/plugins/acs/README.md

```diff
@@ -45,7 +45,7 @@ acs/deployment-name: "deployment-1,deployment-2,deployment-3"
 
 Here's what that will look like in action:
 
-# Example catalog-info.yaml entity definition file
+### Example catalog-info.yaml entity definition file
```

Comment:
As this PR now changes a file that is shipped in the npm package, you would need a changeset